From 8159215331af95bbcb4c23f040c9f69dc858ae26 Mon Sep 17 00:00:00 2001
From: Jair Gonzalez <jair.de.jesus.gonzalez.plascencia@intel.com>
Date: Thu, 4 Jul 2019 19:22:39 -0500
Subject: [PATCH 2/3] Update references to old highgui.h

Signed-off-by: Jair Gonzalez <jair.de.jesus.gonzalez.plascencia@intel.com>
---
 snd2png/snd2png.cpp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/snd2png/snd2png.cpp b/snd2png/snd2png.cpp
index 269799c9..f2e63b79 100644
--- a/snd2png/snd2png.cpp
+++ b/snd2png/snd2png.cpp
@@ -6,10 +6,13 @@
 */

 #include <fftw3.h>
-#include <highgui.h>
 #include <locale.h>
 #include <math.h>
 #include <memory.h>
+#include "opencv2/core/core_c.h"
+#include "opencv2/core/core.hpp"
+#include "opencv2/highgui/highgui_c.h"
+#include "opencv2/highgui/highgui.hpp"
 #include <opencv2/opencv.hpp>
 #include <sndfile.h>
 #include <stdio.h>
--
2.22.0

