From 3666605e15e2f955f290d8379a931171637e758d Mon Sep 17 00:00:00 2001
From: Jair Gonzalez <jair.de.jesus.gonzalez.plascencia@intel.com>
Date: Tue, 9 Jul 2019 17:31:51 -0500
Subject: [PATCH 3/3] videoencoder: Use CXXFLAGS instead of CFLAGS

Signed-off-by: Jair Gonzalez <jair.de.jesus.gonzalez.plascencia@intel.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 10e861a2..b6cb56c6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -92,7 +92,7 @@ pkglibexec_FOLDERS = \
 pkglibexec_PROGRAMS = videoencoder

 videoencoder_SOURCES = videoencoder.cpp
-videoencoder_CFLAGS = -O2 -Wall -W $(OPENCV_CFLAGS)
+videoencoder_CXXFLAGS = -O2 -Wall -W $(OPENCV_CFLAGS)
 videoencoder_LDADD = $(OPENCV_LIBS) $(THEORAENC_LIBS) -lm

 EXTRA_DIST = \
--
2.22.0

